src/rez/utils/filesystem.py

```diff
@@ -340,8 +340,12 @@ def is_subdirectory(path_a, path_b):
     """Returns True if `path_a` is a subdirectory of `path_b`."""
     path_a = os.path.realpath(path_a)
     path_b = os.path.realpath(path_b)
-    relative = os.path.relpath(path_a, path_b)
-    return (not relative.startswith(os.pardir + os.sep))
+    try:
+        relative = os.path.relpath(path_a, path_b)
+    except ValueError:
```

Comment:
what's this about..?

Reply:
Just added a comment.
It's Windows thing, when two paths are on different mount drives, e.g. :
```python
os.path.relpath("c:/hello", "d:/hello")
# ValueError: path is on mount 'c:', start on mount 'd:'
```